Dockerfile

```diff
@@ -44,7 +44,12 @@ RUN bundle config set --local path "vendor/bundle"
 COPY --from=build $RAILS_ROOT $RAILS_ROOT/
 
 EXPOSE 3000
+RUN chmod +x ./entrypoint.sh
+ENTRYPOINT ["./entrypoint.sh"]
```

Comment:
The build fails because this file doesn't exist.

Reply:
Oops! i forgot to git add it.